samples/cpp/speech_sample/utils.hpp

```diff
@@ -405,3 +405,43 @@ std::vector<std::string> convert_str_to_vector(std::string str) {
     }
     return blobName;
 }
+
+/**
+ * @brief Parse layout string like "input0[value0],input1[value1]" or "[value]" (applied to all inputs)
+ * @param layout_string input names with layout values
+ * @param input_info reference to vector of inputs
+ * @return map of inputs with layout values
+ */
+std::map<std::string, std::string> parse_input_layouts(const std::string& layout_string,
```

Comment:
Can we put this function into the common place for sampels and reuse here in benchmark_app to avoid code duplication?

Reply:
Yes, we could, but this version of the function is different from the version of the function in the benchmark. In this case, a different std::vector<ov::Output<ov::Node>> parameter is used and the return value is also different.